pkg/pods/manifest.go

```diff
@@ -191,9 +192,12 @@ func (manifest *Manifest) PlatformConfigFileName() (string, error) {
 // Returns the entity that signed the manifest, if any. If there was no
 // signature, both returns are nil. If the signer is not in the given keyring,
 // an openpgp.ErrUnknownIssuer will be returned.
-func (manifest *Manifest) Signer(keyring openpgp.KeyRing) (*openpgp.Entity, error) {
-	if manifest.signature == nil {
+func (manifest *Manifest) Signer(verifier auth.SigVerifier) (*openpgp.Entity, error) {
+	if manifest.signature == nil || verifier == nil {
```

Comment:
`auth.SigVerifier` is an interface, so it's actually much worse than Mike points out - https://groups.google.com/forum/#!topic/golang-nuts/QzVDKv7p0Vs